### DISCUSSION TITLE
Bintray sunsetting

### DISCUSSION THREAD

**Discussion by hugomcconnell** [reportportal/reportportal#1367] [General]
Hi Guys,

It seems they are sunsetting bintray (http://dl.bintray.com/epam/reportportal) - going forward, where will the project be hosted?

Thanks

**Comment by hugomcconnell** [comment]
ignore this please - I see the answer in the issues section

**Comment by evjlobanova** [comment]
The progress on this problem you can track https://github.com/reportportal/reportportal/issues/1320
